serving/docker/partition/properties_manager.py

```diff
@@ -31,14 +31,13 @@
 
 class PropertiesManager(object):
 
-    def __init__(self, properties_dir):
-        self.properties = {}
-        self.properties_dir = properties_dir
+    def __init__(self, properties):
+        self.properties = properties
+        self.properties_dir = properties.get('properties_dir')
         self.entry_point_url = None
 
-        self.load_properties()
+        load_properties(self.properties_dir, self.properties)
```

Comment:
the command line option should override `serving.properties`, this implementation will override command line